.github/ISSUE_TEMPLATE/REPORT_A_BUG.yml

```diff
@@ -0,0 +1,76 @@
+name: '🐛 Report a bug'
+description: 'Is something not working as you expect?'
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to file a bug report!
+  - type: textarea
+    id: reproduce-bug
+    attributes:
+      label: 'What steps are needed to reproduce the bug?'
+      placeholder: |
+        e.g. "CLI with `stylelint '**/*.css' --config myconfig.json`"
+    validations:
+      required: true
+  - type: input
+    id: stylelint-rule
+    attributes:
+      label: 'Which rule, if any, is the bug related to?'
+      placeholder: 'e.g. `number-leading-zero`'
+    validations:
+      required: false
```

Comment:
Let's lose the `stylelint-rule` field as it's redundant if we ask for the smallest reproducible config.

Reply:
Do you mean that we should delete this field?

Reply:
Done. c128ec66

I think the `stylelint-rule` field is no longer needed from [the comment](https://github.com/stylelint/stylelint/pull/5678/files#r739686521).